modules/exploits/linux/http/cisco_rv32x_rce.rb

```diff
@@ -0,0 +1,164 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HttpServer::HTML
+  include Msf::Exploit::CmdStager
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Cisco RV320 and RV325 Unauthenticated Remote Code Execution",
+      'Description'    => %q{
+This exploit module combines an information disclosure (CVE-2019-1653)
+and a command injection vulnerability (CVE-2019-1652) together to gain
+unauthenticated remote code execution on Cisco RV320 and RV325 small business
+routers. Can be exploited via the WAN interface of the router. Either via HTTPS
+on port 443 or HTTP on port 8007 on some older firmware versions.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         => [ 'RedTeam Pentesting GmbH','Benjamin Grap' ],
+      'References'     => [
+          [ 'CVE','2019-1653' ],
+          [ 'CVE','2019-1652' ],
+          [ 'EDB','46243' ],
+          [ 'BID','106728' ],
+          [ 'BID','106732' ],
+          [ 'URL', 'https://www.redteam-pentesting.de/en/advisories/rt-sa-2018-002/-cisco-rv320-unauthenticated-configuration-export' ],
+          [ 'URL', 'https://www.redteam-pentesting.de/en/advisories/rt-sa-2018-004/-cisco-rv320-command-injection' ]
+      ],
+      'Platform'       => 'linux',
+      'Targets'        =>
+        [
+         [ 'LINUX MIPS64',
+          {
+           'Platform' => 'linux',
+           'Arch'     => ARCH_MIPS64
+          }
+         ]
+        ],
+      'Payload'        =>
+        {
+         'BadChars' => ""
+        },
+      'CmdStagerFlavor' => [ 'bourne' ],
+      'Privileged'     => false,
```

Comment:
should be set to "True", the exploit gives a "uid=0" (root) shell (albeit with a gid of 99, but that doesn't matter).

Reply:
According to the documentation this flag is not about the kind of privilege you get, but what kind of privilege is required to run the module. See https://www.rubydoc.info/github/rapid7/metasploit-framework/Msf%2FModule:privileged